Sources/MapboxMaps/Gestures/GestureHandlers/DoubleTapToZoomInGestureHandler.swift

```diff
@@ -1,11 +1,15 @@
 import UIKit
 
+internal protocol DoubleTapToZoomInGestureHandlerProtocol: GestureHandler {
```

Comment:
I don't think that the protocol name should say `DoubleTap` as there is nothing double-tap-related in it. 
Should it be just `ZoomGestureHandler`? 🤔